docs/Building-Tachyon-Master-Branch.md

```diff
@@ -24,67 +15,87 @@ The prerequisite for this guide is that you have [Java 6 (or above)](Java-Setup.
-system. If this is the first time you are building the project, it can take a while to download all
-the dependencies. Subsequent builds, however, will be much faster.
+The Maven build system fetches its dependencies, compiles source code, runs unit tests, and packages 
+the system. If this is the first time you are building the project, it can take a while to download 
+all the dependencies. Subsequent builds, however, will be much faster.
 
-Once it is built, you can start Tachyon:
+Once Tachyon is built, you can start it with:
 
-    $ cp conf/tachyon-env.sh.template conf/tachyon-env.sh
-    $ ./bin/tachyon format
-    $ ./bin/tachyon-start.sh local
+```bash
+$ cp conf/tachyon-env.sh.template conf/tachyon-env.sh
+$ ./bin/tachyon format
+$ ./bin/tachyon-start.sh local
+```
 
 To verify that Tachyon is running, you can visit [http://localhost:19999](http://localhost:19999) or
-check the log in the folder tachyon/logs. You can also run a simple program:
+check the log in the `tachyon/logs` directory. You can also run a simple program:
 
-    $ ./bin/tachyon runTest Basic CACHE_THROUGH
+```bash
+$ ./bin/tachyon runTest Basic CACHE_THROUGH
+```
 
 You should be able to see results similar to the following:
 
-    /default_tests_files/BasicFile_CACHE_THROUGH has been removed
-    2015-09-06 13:27:17,358 INFO   (MasterClient.java:connect) - Tachyon client (version 0.8.0-SNAPSHOT) is trying to connect with master @ localhost/127.0.0.1:19998
-    2015-09-06 13:27:17,375 INFO   (MasterClient.java:connect) - User registered with the master @ localhost/127.0.0.1:19998; got UserId 9
-    2015-09-06 13:27:17,380 INFO   (BasicOperations.java:createFile) - createFile with fileId 5 took 25 ms.
-    2015-09-06 13:27:17,397 INFO   (WorkerClient.java:connect) - Trying to get local worker host : 10.239.44.23
-    2015-09-06 13:27:17,405 INFO   (WorkerClient.java:connect) - Connecting local worker @ /10.239.44.23:29998
-    2015-09-06 13:27:17,438 INFO   (BlockOutStream.java:get) - Writing with local stream. tachyonFile: /default_tests_files/BasicFile_CACHE_THROUGH, blockIndex: 0, opType: CACHE_THROUGH
-    2015-09-06 13:27:17,451 INFO   (FileUtils.java:createStorageDirPath) - Folder /mnt/ramdisk/tachyonworker/9 was created!
-    2015-09-06 13:27:17,454 INFO   (LocalBlockOutStream.java:<init>) - /mnt/ramdisk/tachyonworker/9/5368709120 was created! tachyonFile: /default_tests_files/BasicFile_CACHE_THROUGH, blockIndex: 0, blockId: 5368709120, blockCapacityByte: 536870912
-    2015-09-06 13:27:17,468 INFO   (BasicOperations.java:writeFile) - writeFile to file /default_tests_files/BasicFile_CACHE_THROUGH took 87 ms.
-    2015-09-06 13:27:17,470 INFO   (BlockInStream.java:get) - Reading with local stream.
-    2015-09-06 13:27:17,564 INFO   (BasicOperations.java:readFile) - readFile file /default_tests_files/BasicFile_CACHE_THROUGH took 96 ms.
-    Passed the test!
+```bash
+/default_tests_files/BasicFile_CACHE_THROUGH has been removed
```

Comment:
When it is run for the first time, the info should not be like this since the file hasn't been in Tachyon.